pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -488,12 +489,13 @@ self: super: {
   });
 
   himalaya-vim = super.himalaya-vim.overrideAttrs (old: {
-    postPatch = ''
-      substituteInPlace plugin/himalaya.vim \
-        --replace "if !executable('himalaya')" "if v:false"
-      substituteInPlace autoload/himalaya/request.vim \
-        --replace "'himalaya" "'${himalaya}/bin/himalaya"
-    '';
+    buildInputs = [ himalaya ];
```

Comment:
This all looks good though. Great to see we don't have to patch the package to get those boolean checks now!